cmd/tegola/cmd/root.go

```diff
@@ -80,7 +81,9 @@ func initConfig(configFile string, cacheRequired bool) (err error) {
 	if len(conf.Cache) == 0 && cacheRequired {
 		return fmt.Errorf("No cache defined in config, please check your config (%v).", configFile)
 	}
-	if len(conf.Cache) > 0 {
+	if serverNoCache {
+		log.Info("Turn off the cache.")
```

Comment:
Let's change this log to "Cache explicitly turned off by user" or something a bit more explicit.